SA/design_concept/content/poc/bolero.typ

```diff
@@ -0,0 +1,15 @@
+#import "../../../acronyms.typ": ac
+
+= Bolero
+Bolero is an F\# wrapper around Blazor #footnote("https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor"), leveraging different libraries to create web applications.
+With Blazor, web applications can run both on server side or on client-side using WebAssembly.
```

Comment:
```suggestion
With Blazor, web applications can run both on server-side through web-sockets or on client-side through WebAssembly.
```

I think that is the case, but the description makes it seem like it runs on the server through webassembly as well